index.html

```diff
@@ -3007,69 +3007,299 @@ <h3>Truncating or limiting the length of strings</h3>
+        
+        <img src="images/indivisible-memorable.jpg">
+        
+        <p>Some scripts make the difference between visual text units and pixel size even more obvious. For example, here is the word for "Unicode" in Tamil:</p>
+        
+        <p class="bigtext">யூனிக்கோடு</p>
+        
+        <p>Tamil, like many South Asian scripts, is written in syllables. Each syllable is treated by Tamil speakers as a [=visual text unit=]. The word for "Unicode" in Tamil is a transliteration of the English word. If we divide the syllables up, as shown below, we can see that some are composed of multiple shapes and consume far more space than others:</p>
+        
+        <table>
+            <tr>
+                <th>Syllable</th>
+                <td class="bigtext" style="width:6rem;text-align:center">யூ</td>
+                <td class="bigtext" style="width:6rem;text-align:center">னி</td>
+                <td class="bigtext" style="width:6rem;text-align:center">க்</td>
+                <td class="bigtext" style="width:6rem;text-align:center">கோ</td>
+                <td class="bigtext" style="width:6rem;text-align:center">டு</td>
+            </tr>
+            <tr>
+                <th>IPA</th>
+                <td style="width:6rem;text-align:center">juː</td>
+                <td style="width:6rem;text-align:center">ni</td>
+                <td style="width:6rem;text-align:center">k</td>
+                <td style="width:6rem;text-align:center">koː</td>
+                <td style="width:6rem;text-align:center">ʈu</td>
+            </tr>
+        </table>
 
-    <p>If the limitation is related to the number of display positions, the grapheme count usually corresponds most closely to the expected limit. Note that proportional width fonts, combining marks, complex scripts, and many other factors complicate counting "screen positions". In Web pages, for example, the CSS <kbd>text-overflow</kbd> property provides visual truncation without disturbing the content of the text. Attempts to estimate the size of a given piece of text based on the number of Unicode code points or even the number of grapheme clusters is mostly futile.</p>
+    </aside>
+    
+    <p>Most limits are actually related to storage limitations, such as the size of a database field or a length limit in a protocol. Such limits are expressed in terms of [=code points=] in Unicode or [=code units=] (such as bytes) in a specific [=character encoding=]. Code points provides the best user experience, since all Unicode code points are treated identically: if text is truncated after 40 code points, all languages and scripts get the same number of code points to work with. By contrast, when the size limit is expressed in code units such as bytes in UTF-8, users who write in a language that mostly uses ASCII letters get many more characters (code points) for a given size limit than user's whose language is mostly made up of characters that take 2-, 3-, or 4-bytes per code point.</p>
```

Comment:
We seem to be missing some mustard here, along the lines of "When limits are related to storage limitations, express them using code points."